lib/compilers/ppci.js

```diff
@@ -0,0 +1,29 @@
+
+const Compile = require('../base-compiler'),
+    utils = require('../utils');
+
+function compilePpci(info, env) {
+    const compiler = new Compile(info, env);
+
+    compiler.runCompiler = function (compiler, options, inputFilename, execOptions) {
+        if (!execOptions) {
+            execOptions = compiler.getDefaultExecOptions();
+        }
+
+        compiler = 'python3';
```

Comment:
`const` variable being reasigned. You probably want to create a new variable here

Reply:
Yes, indeed.